test/LiquidityPool.tests.ts

```diff
@@ -341,6 +337,7 @@ describe("LiquidityPoolTests", function () {
             let toChainId = 1;
 
             await addTokenLiquidity(tokenAddress, liquidityToBeAdded, owner);
+            await executorManager.addExecutor(executor.address);
```

Comment:
Why is this needed in this function?

Reply:
For the sendFundsToUser call on line 371

Reply:
I removed the addExecutor call in beforeEach, since other tests were calling the addExecutor manually